types/trust__keyto/index.d.ts

```diff
@@ -0,0 +1,133 @@
+// Type definitions for @trust/keyto 1.0
+// Project: https://github.com/EternalDeiwos/keyto#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = Key;
+
+declare class Key {
+    /**
+     * Import a key.
+     *
+     * @param format The format of the key.
+     * - `'pem'`: will parse a PEM encoded string (as per OpenSSL output).
+     * - `'jwk'`: will parse a JWK object or JSON String
+     * - `'blk'`: will parse a hex encoded key string as used on various blockchains (limited to secp256k1 keys).
+     *
+     * @example <caption>Decode PEM and convert to JWK</caption>
+     * import keyto = require('@trust/keyto')
+     * import * as assert from 'node:assert'
+     *
+     * const pemPrivate = getPrivatePemStringSomehow()
+     * const jwk = getPublicJwkSomehow()
+     *
+     * // String data can either be passed in directly:
+     * const key = keyto.from(pemPrivate, 'pem').toJwk('public')
+     *
+     * assert.equal(jwk, key)
+     *
+     * @example <caption>Decode HEX (Blockchain) private key and convert to PEM PKCS8 public key</caption>
+     * import keyto = require('@trust/keyto')
+     * import * as assert from 'node:assert'
+     *
+     * const blk = getPrivateBlockchainHexStringSomehow()
+     * const pemPublic = getPublicPemSomehow()
+     *
+     * const key = keyto.from(blk, 'blk').toString('pem', 'public_pkcs8')
+     *
+     * assert.equal(pemPublic, key)
+     */
+    static from(key: Key.JWK, format: 'jwk'): Key;
+    static from(key: string, format: Key.SerializableFormat): Key;
+
+    /**
+     * A high level class for accepting and processing keys.
+     */
+    constructor(key: Key.JWK | string | Buffer, options: Key.KeyOptionsWithCrv | Key.KeyOptionsWithOid);
```

Comment:
should this one be hidden (protected/private/abstract/etc), as the api uses factory methods and instance methods, not constructor.
Also this line:
https://github.com/EternalDeiwos/keyto/blob/master/src/index.js#L74

Reply:
done